### ISSUE TITLE
test peak_anno_2 and peak_anno_4 are failing (at least under OSX).

### ISSUE THREAD

**Issue by dputhier** [dputhier/pygtftk#60]
Below is the result of peak_anno_01:

cat peak_annotation/00_peak_anno_stats_* | grep gene

         cat peak_annotation/00_peak_anno_stats_* | grep gene
         gene	14.97	1.98	0.60119	0.096	16	0.33299999999999996	61.35	32.94	0.8227700000000001	0.28983000000000003	75	0.0389

Also note that, in peak_anno_01, you should better call gtftk peak_anno using the command-wise argument  **-D (--no-date)**. It would then produce 00_peak_anno_stats.tsv compare to 00_peak_anno_stats_20190308-174721.tsv. It would allow you to write the file name without the wildcard:

 cat peak_annotation/00_peak_anno_stats.tsv | grep gene

**Comment by dputhier** [comment]
The command of peak_anno_04:

       cat peak_annotation/00_peak_anno_stats_* | grep gene | cut -f 11

returns:

0.28983000000000003

**Comment by qferre** [comment]
Whoops. My bad. I fixed the order in which the peak_anno_stats columns are written to account for certain special cases and forgot to update the unitary tests.

I swear, sometimes I think I'd forget my own head if it wasn't attached to my shoulders :)

I fixed my stupid mistake in b0e75f420326d9ceef00db0bfb6ec9c7fa98e78b.

I ran all the tests using `make nose` and `make bats_cmd CMD=peak_anno; make clean`, and they are all passing for me (except for the gtf_interface.py one, but that's another problem you already know about).
